Makefile

```diff
@@ -114,7 +114,7 @@ re: clean all
 #---------------------------------------------------------------------------------
 
 test: all
-	@cp $(OUTPUT).3gx $(CITRA_DIR)/spicerack/bin/
+	@cp $(OUTPUT).3gx $(CITRA_DIR)/spicerack/bin/Saltwater.3gx
```

Comment:
replace with `$(OUTPUT).3gx`

Reply:
this was intended, barista is hard coded to look for `/spicerack/bin/Saltwater.3gx`, not `$(OUTPUT).3gx`.

I have my fork under `SaltwaterMyFork` which means it will generate a  `SaltwaterMyFork.g3x`